apps/site/src/components/Footer/Footer.module.scss

```diff
@@ -0,0 +1,20 @@
+@use "@/lib/styles/variables.scss" as variables;
+
+.footer {
+	padding-bottom: 100px;
```

Comment:
For this and other `px` values related to padding, margin, and font size (non exhaustive list), it might be better to use relative length units (i.e. `em`, `rem`, etc.).

See https://developer.mozilla.org/en-US/docs/Learn/CSS/Building_blocks/Values_and_units